casper/src/test/scala/coop/rchain/casper/HashSetCasperTest.scala

```diff
@@ -875,18 +876,20 @@ class HashSetCasperTest extends FlatSpec with Matchers {
     val pkStr                    = Base16.encode(pk)
     val amount                   = 314L
     val forwardCode              = BondingUtil.bondingForwarderDeploy(pkStr, pkStr)
+    val bondingCode              = BondingUtil.faucetBondDeploy[Task](amount, "ed25519", pkStr, sk).unsafeRunSync
```

Comment:
This method implicitly relies on `RuntimeManager[F]` which in this test is a `RuntimeManager[Task]`. I had to move this out of the for comprehension, because the context used there is `Effect`.

Do not want to resolve this now, as it will cause the PR to bloat